src/proper_sa.erl

```diff
@@ -109,8 +108,8 @@
 %% number of the search steps and the strategy.
 
 %% @private
--spec init_strategy(proper:setup_opts()) -> sa_data().
-init_strategy(#{numtests := Steps}) ->
+-spec init_strategy(pos_integer()) -> sa_data().
```

Comment:
Better introduce a type for this (in some target-related file and use throughout)
```erlang
-type search_steps() :: pos_integer().
```